test/SlowTests/Issues/RavenDB_17626.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using FastTests;
+using Raven.Client.Documents.Indexes;
+using Raven.Server.Documents.Indexes.Static.Extensions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues;
+
+public class RavenDB_17626 : RavenTestBase
+{
+    public RavenDB_17626(ITestOutputHelper output) : base(output)
+    {
+    }
+
+    [Fact]
+    public void Should_Work()
```

Comment:
Can we have meaningful test name here? I don't like this convention of calling test "should work". Then it's hard to figure what is it about when it fails on Jenkins

Reply:
Oh sorry, I forgot to change it at the end. Fixed now.